diskv.go

```diff
@@ -98,13 +142,32 @@ func New(o Options) *Diskv {
 	return d
 }
 
+// convertToAdvancedTransform takes a classic Transform function and
+// converts it to the new AdvancedTransform
+func convertToAdvancedTransform(oldFunc func(s string) []string) AdvancedTransformFunction {
+	return func(s string) *PathKey {
+		return &PathKey{Path: oldFunc(s), FileName: s}
+	}
+}
+
 // Write synchronously writes the key-value pair to disk, making it immediately
 // available for reads. Write relies on the filesystem to perform an eventual
 // sync to physical media. If you need stronger guarantees, see WriteStream.
 func (d *Diskv) Write(key string, val []byte) error {
 	return d.WriteStream(key, bytes.NewReader(val), false)
 }
 
+// WriteString writes a string key-value pair to disk
+func (d *Diskv) WriteString(key string, val string) error {
```

Comment:
Why the new method? Does it implement some interface?

Reply:
This is very convenient. In my code that uses this library I was always casting to []byte and back, very annoying. This is a similar pattern to `bytes.Buffer`